<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="welcome_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(29, 155, 240)</string>
   </property>
   <property name="text">
    <string>Image Recognisation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
